src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ExternalMethodologyViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+public record ExternalMethodologyViewModel
+{
+    public string Title { get; init; } = string.Empty;
+
+    public string Url { get; init; } = string.Empty;
+
+    public ExternalMethodologyViewModel()
+    {
+    }
+
+    public ExternalMethodologyViewModel(ExternalMethodology externalMethodology)
+    {
+        Title = externalMethodology.Title;
+        Url = externalMethodology.Url;
+    }
```

Comment:
This looks like it's been cloned from `Content.Services.ViewModels` but I don't think this copy constructor for `ExternalMethodology` is used anywhere by Admin?

See my other comment about how this could be used though 🙂 .